src/content/reference/hi/p5.Shader/copyToContext.mdx

```diff
@@ -1,60 +1,17 @@
 ---
 title: copyToContext
-module: 3D
-submodule: Material
+module: 3डी
+submodule: भौतिक
```

Comment:
Can we use a simpler translation for material?

Reply:
For Material, i have 3 suggestions - 
- सामग्री
- पदार्थ
- द्रव्यमान

In my opinion, द्रव्यमान seems most suited for the given context. @TanviKumar ?